Apps/GatewayApi/src/Models/CommunicationModel.cs

```diff
@@ -0,0 +1,76 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.GatewayApi.Models
+{
+    using System;
+    using System.Text.Json.Serialization;
+    using HealthGateway.Common.Data.Constants;
+    using CommunicationStatus = HealthGateway.GatewayApi.Constants.CommunicationStatus;
+    using CommunicationType = HealthGateway.GatewayApi.Constants.CommunicationType;
+
+    /// <summary>
+    /// Model that provides a user representation of a communication database model.
+    /// </summary>
+    public class CommunicationModel
```

Comment:
When we refactor for our V2 endpoints, most of the properties in here can be removed as they are not relevant in this context.

Reply:
ok.